project-base/storefront/components/Pages/ProductDetail/ProductDetailTabs/ProductDetailTabs.tsx

```diff
@@ -40,30 +40,78 @@ export const ProductDetailTabs: FC<ProductDetailTabsProps> = ({ description, par
+                        .filter(parameter => parameter.group === null)
+                        .sort((a, b) => a.name.localeCompare(b.name))
+                        .length > 0 && (
+                        <div>
+                            <Table className="border-0 p-0 max-w-screen-lg mx-auto">
+                                {parameters
+                                    .filter(parameter => parameter.group === null)
+                                    .sort((a, b) => a.name.localeCompare(b.name))
+                                    .map((parameter) => (
+                                        <Row
+                                            key={parameter.uuid}
+                                            className="bg-tableBackground odd:bg-tableBackgroundContrast border-none"
+                                        >
+                                            <Cell className="py-2 text-left text-sm font-bold uppercase leading-5">
+                                                {parameter.name}
+                                            </Cell>
+                                            <Cell className="py-2 text-right text-sm leading-5">
+                                                {parameter.values.map((value, index) =>
+                                                    formatParameterValue(
+                                                        value.text + (parameter.unit?.name ? ` (${parameter.unit.name})` : ''),
+                                                        index
+                                                    )
+                                                )}
+                                            </Cell>
+                                        </Row>
+                                    ))}
+                            </Table>
+                        </div>
+                    )}
 
-                                <Cell className="py-2 text-right text-sm leading-5">
-                                    {parameter.values.map((value, index) =>
-                                        formatParameterValue(
-                                            value.text + (parameter.unit?.name ? ` (${parameter.unit.name})` : ''),
-                                            index,
-                                        ),
-                                    )}
-                                </Cell>
-                            </Row>
-                        ))}
-                    </Table>
+                    {Object.entries(
+                        parameters.reduce((acc, parameter) => {
+                            if (parameter.group) {
+                                if (!acc[parameter.group]) {
+                                    acc[parameter.group] = [];
+                                }
+                                acc[parameter.group].push(parameter);
+                            }
+                            return acc;
+                        }, {})
```

Comment:
The same applies here. Please move complex logic to the body of the component instead. Also, did you setup linter in your IDE correctly? I can see bunch of type errors when checking your code locally 
![image](https://github.com/user-attachments/assets/4eb20dc5-815f-4d16-b579-2eb0cc7ef7a1)